model/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala

```diff
@@ -412,11 +420,11 @@ trait ExecutionJsonSupport extends JsonSupport {
 
   implicit val ExternalEntityInfoFormat = jsonFormat2(ExternalEntityInfo)
 
-  implicit val SubmissionFormat = jsonFormat15(Submission)
+  implicit val SubmissionFormat = jsonFormat16(Submission)
```

Comment:
What do these numbers after jsonFormat mean?

Reply:
The number represents how many parameters are there in the case class